src/OpenTelemetry/OpenTelemetry.csproj

```diff
@@ -19,6 +19,7 @@
     <PackageReference Include="System.Reflection.Emit.Lightweight" Version="$(SystemReflectionEmitLightweightPkgVer)" Condition="'$(TargetFramework)' != 'net6.0'" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsLoggingPkgVer)" />
     <PackageReference Include="Microsoft.Extensions.Logging.Configuration" Version="$(MicrosoftExtensionsLoggingConfigurationPkgVer)" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="$(MicrosoftExtensionsConfigurationEnvironmentVariablesPkgVer)" />
```

Comment:
Don't love adding this dependency, but we do make heavy use of environment variables so it didn't seem totally unreasonable?

Reply:
I'm not personally super concerned, but I have heard concern from end users regarding dependency bloat... 

We could experiment with breaking out SDK configuration into it's own package. This is _kind of_ similar to what Java has with their auto-config module https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure-spi